src/pages/home/report/ReportActionsList.js

```diff
@@ -145,6 +145,8 @@ function ReportActionsList({
     const opacity = useSharedValue(0);
     const userActiveSince = useRef(null);
     const unreadActionSubscription = useRef(null);
+    const readNewestActionSubscription = useRef(null);
```

Comment:
Why are we using refs, if we don't access `.current` outside of a single anonymous function? Does this bring any value over using local `const`s?

```ts
const unreadActionSubscription = DeviceEventEmitter.addListener(...);
/// etc.
```